gen3/bin/kube-setup-argo.sh

```diff
@@ -243,7 +243,7 @@ if [[ "$ctxNamespace" == "default" || "$ctxNamespace" == "null" ]]; then
 
     helm repo add argo https://argoproj.github.io/argo-helm --force-update 2> >(grep -v 'This is insecure' >&2)
     helm repo update 2> >(grep -v 'This is insecure' >&2)
-    helm upgrade --install argo argo/argo-workflows -n argo -f ${valuesFile}
+    helm upgrade --install argo argo/argo-workflows -n argo -f ${valuesFile} --version 0.22.11
```

Comment:
It would be nice if we could set that as the default but allow it to be overwritten, similar to [this](https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-karpenter.sh#L19).